disperser/dataapi/server.go

```diff
@@ -586,6 +599,36 @@ func (s *server) FetchDeregisteredOperators(c *gin.Context) {
 	})
 }
 
```

Comment:
Do we need another API which takes the socket as parameter? That could be used by CLI *before* registration.

This `OperatorPortCheck` will still be useful for creating a metric at Node to check reachability periodically.

Reply:
Adding generic socket port check opens the door to it being abused as a general purpose penetration testing service. Forcing operator_id avoids a swath of potential attacks on the endpoint/dataapi in general.